Regulator_Linear.lib

```diff
@@ -581,6 +581,51 @@ X VI 3 -300 0 100 R 50 50 1 1 W
 ENDDRAW
 ENDDEF
 #
+# BD00FC0WEFJ
+#
+DEF BD00FC0WEFJ U 0 20 Y Y 1 F N
+F0 "U" 0 300 50 H V C CNN
+F1 "BD00FC0WEFJ" 0 200 50 H V C CNN
+F2 "Package_SO:HTSOP-8-1EP_3.9x4.9mm_P1.27mm_EP2.4x3.2mm" -400 100 50 H I C CNN
+F3 "" 100 0 50 H I C CNN
+$FPLIST
+ HTSOP*1EP*3.9x4.9mm*P1.27mm*
+$ENDFPLIST
+DRAW
+S -200 -200 200 50 0 1 10 f
+X VO 1 300 0 100 L 50 50 1 1 w
+X FB 2 300 -100 100 L 50 50 1 1 I
+X GND 3 0 -300 100 U 50 50 1 1 W
+X GND 4 0 -300 100 U 50 50 1 1 P N
+X EN 5 -300 -100 100 R 50 50 1 1 I
+X GND 6 0 -300 100 U 50 50 1 1 P N
+X GND 7 0 -300 100 U 50 50 1 1 P N
+X VCC 8 -300 0 100 R 50 50 1 1 W
+X GND 9 0 -300 100 U 50 50 1 1 P N
+ENDDRAW
+ENDDEF
+#
+# BD00FC0WFP
+#
+DEF BD00FC0WFP U 0 20 Y Y 1 F N
+F0 "U" 0 300 50 H V C CNN
+F1 "BD00FC0WFP" 0 200 50 H V C CNN
+F2 "Package_TO_SOT_SMD:TO-263-5_TabPin6" 0 100 50 H I C CNN
```

Comment:
package must be TO-252-5
![εικόνα](https://user-images.githubusercontent.com/48727784/94730841-835a0c00-036c-11eb-9c8e-5e5bd6601854.png)
There is no 263 at the ds. Please fix those parts that are wrong (if others) and adjust filters.